tests/pipelines/bicep-build-to-validate.yml

```diff
@@ -245,6 +245,23 @@ jobs:
       script: |
         az deployment group create --resource-group $(ResourceGroupName) --template-file infra-as-code/bicep/modules/vnetPeering/vnetPeering.bicep --parameters @infra-as-code/bicep/modules/vnetPeering/parameters/vnetPeering.parameters.min.json parDestinationVirtualNetworkId="/subscriptions/$(subscriptionId)/resourceGroups/$(ResourceGroupName)/providers/Microsoft.Network/virtualNetworks/vnet-spoke" parSourceVirtualNetworkName="alz-hub-eastus" parDestinationVirtualNetworkName="vnet-spoke"
 
+  # Verify that WhatIf does not found differences between code and environment
+  - task: Bash@3     
+    displayName: Az CLI After Deployment What-If Management Groups for PR
+    name: whatif_mgs
+    condition: and(or(ne(variables['gitManagementOUTPUT'], ''), ne(variables['gitLoggingOUTPUT'], ''), ne(variables['gitSpokeOUTPUT'], ''), ne(variables['gitHubOUTPUT'], ''), ne(variables['gitVwanOUTPUT'], ''), ne(variables['gitVwanNwcOUTPUT'], ''), ne(variables['gitVnetPeerOUTPUT'], '')), ne(variables['subscriptionId'], ''))
+    inputs:
+      targetType: 'inline'
+      script: |
+        tenantRootGroupID=$(az account management-group list | jq '.[] | select(.displayName | test("Tenant Root Group")) | .id' | tr -d '"')
```

Comment:
@olljanat why the `test` part in here?

Reply:
Perhaps because I'm not that familiar with jq syntax and have copied some example from internet 😄

Reply:
Here is what i would use instead `tenantRootGroupID=$(az account management-group list | jq '.[] | select(.displayName=="Tenant Root Group") | .id' | tr -d '"')`

Reply:
```suggestion
        tenantRootGroupID=$(az account management-group list | jq '.[] | select(.displayName=="Tenant Root Group") | .id' | tr -d '"')
```